code/modules/nano/nanomapgen.dm

```diff
@@ -57,22 +57,21 @@
 	log_to_dd("NanoMapGen: <B>GENERATE MAP ([startX],[startY],[currentZ]) to ([endX],[endY],[currentZ])</B>")
 	to_chat(usr, "NanoMapGen: <B>GENERATE MAP ([startX],[startY],[currentZ]) to ([endX],[endY],[currentZ])</B>")
 
-	var/count = 0;
 	for(var/WorldX = startX, WorldX <= endX, WorldX++)
```

Comment:
since you're using CHECK_TICK, you'll probably want this to have set waitfor = 0, because of the sleeps